### DISCUSSION TITLE
saf-training/installation/WindowsInstall

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#45] [Ideas]
# saf-training/installation/WindowsInstall

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/installation/WindowsInstall.html

**Comment by jaimeporras** [comment]
The Windows installation worked perfectly with one exception: I had to reboot my system before I could verify inspec was installed properly with said command $ inspec version
